oak-jcr/src/main/java/org/apache/jackrabbit/oak/jcr/xml/ImporterImpl.java

```diff
@@ -336,138 +339,150 @@ public void startNode(@NotNull NodeInfo nodeInfo, @NotNull List<PropInfo> propIn
             throws RepositoryException {
         Tree parent = parents.peek();
         Tree tree = null;
-        String id = nodeInfo.getUUID();
```

Comment:
The diff is confusing, but the block has not been removed but just wrapped in a try block to revert changes afterwards.

Reply:
What I sometimes do in such cases (to simplify the diff) is create a new method, e.g. getPropertyImportersTry, and then call it in a try-catch.

But I don't know this area so I'm afraid I'm not the right person to review...

Reply:
> But I don't know this area so I'm afraid I'm not the right person to review...

Who is the right person then?